apps/admin-gui/src/assets/i18n/en.json

```
@@ -283,6 +283,7 @@
     "MEMBERS": {
       "TITLE": "Members",
       "ADD_MEMBER": "Add",
+      "ADD_MEMBER_DISABLED": "You cannot add members manually in this Vo.",
```

```suggestion
      "ADD_MEMBER_DISABLED": "You cannot add members manually into this Vo.",
```
probably better wording

---

Edited.